bevy_kayak_renderer/src/render/unified/shader.wgsl

```diff
@@ -19,6 +23,7 @@ struct VertexOutput {
     @location(3) size: vec2<f32>,
     @location(4) border_radius: f32,
     @location(5) pixel_position: vec2<f32>,
+    @location(6) _unused: f32,
```

Comment:
I don't think this is required?

Reply:
Gave it a try without - it works! Is the reason for this that `VertexOutput` is only created & used inside the shader, but `QuadType` is provided from outside the shader?